bundle/edu.gemini.shared.util/src/main/scala/edu/gemini/shared/util/DateTimeUtils.scala

```diff
@@ -0,0 +1,86 @@
+package edu.gemini.shared.util
+
+import java.time.{Instant, ZoneId, ZonedDateTime}
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+import java.util.{Comparator, Date, TimeZone}
+import java.util.function.ToLongFunction
+
+import scala.concurrent.duration._
+import scalaz._
+import Scalaz._
+
+object DateTimeUtils {
+  val UTC: ZoneId         = ZoneId.of("UTC")
+  val DefaultZone: ZoneId = ZoneId.systemDefault()
+
+  // Standard UTC formatters.
+  val YYYY_MMM_DD_Formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MMM-dd").withZone(UTC)
+  val YYYY_MMM_DD_HHMM_Formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MMM-dd HH:mm").withZone(UTC)
+  val YYYY_MMM_DD_HHMMSS_Formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MMM-dd HH:mm:ss").withZone(UTC)
+  val YYYY_MMM_DD_HHMMSS_Z_Formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MMM-dd HH:mm:ss z").withZone(UTC)
+  val HHMMSS_Formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss").withZone(UTC)
```

Comment:
Would it make sense to have a `Formatters` case class parameterized by zone, and then instances for UTC and system default? You could then say `DateTimeUtils.Formatters.UTC.YYYY_MMM_DD_HHMMSS_Z` and it would be pretty clear what's happening at the call site. As-is you just kind of have to know that it's UTC.

Reply:
Hmmm. I think I like this idea better. I'll take a look at that.

Reply:
Okay, I parameterized the formatters, and I like it better this way: this forces a time zone to be used and avoids repetitive declarations.